docs/core/versions/binding.md

```diff
@@ -0,0 +1,117 @@
+
+On rare occasions, you may need to use a different version of the SDK. You specify that version in a [global.json file](../tools/global-json.md). The "use latest" policy means you only use `global.json` to specify a .NET Core version earlier than the latest installed version.
+
+`global.json` can be placed anywhere in the file hierarchy. The CLI searches upward from the project directory for the first `global.json` it finds. You control which projects a given `global.json` applies to by its place in the file system. The .NET CLI searches for a `global.json` file iteratively navigating the path upward from the current working directory. The first `global.json` file found specifies the version used. If that version is installed, that version is used. If the SDK specified in the global.json is not found, the .NET CLI rolls forward to the latest SDK installed. This is the same as the default behavior, when no global.json file is found.
+
+The following example shows the `global.json` syntax:
+
+``` json
+{
+  "sdk": {
+    "version": "2.0.0"
+  }
+}
+```
+
+The process for selecting an SDK version is:
+
+- `dotnet` searches for a `global.json` file iteratively reverse-navigating the path upward from the current working directory.
+- `dotnet` uses the SDK specified in the first `global.json` found.
+- `dotnet` binds to the latest installed SDK if no `global.json` is found.
+
+## Target Framework Monikers define build time APIs
+
+You build your project against APIs defined in a **Target Framework Moniker** (TFM). You specify the target framework in the project file. Set the `TargetFramework` element in your project file as shown in the following example:
+
+``` xml
+<TargetFramework>netcoreapp2.0</TargetFramework>
+```
+
+You may build your project against multiple TFMs. Setting multiple target frameworks is more common for libraries but can be done with applications as well. You specify a `TargetFrameworks` property (plural of `TargetFramework`). The target frameworks will be semicolon-delimited as shown in the following example:
+
+``` xml
+<TargetFrameworks>netcoreapp2.0;net47</TargetFrameworks>
+```
+
+A given SDK supports a fixed set of frameworks, capped to the target framework of the runtime it ships with. For example, the .NET Core 2.0 SDK includes the .NET Core 2.0 runtime, which is an implementation of the `netcoreapp2.0` target framework. The .NET Core 2.0 SDK will support `netcoreapp1.0`, `netcoreapp1.1`, and `netcoreapp2.0` but not `netcoreapp2.1` (or higher). You install the .NET Core 2.1 SDK to build for `netcoreapp2.1`
+
+.NET Standard target frameworks are also capped in the same way. The .NET Core 2.0 SDK is capped to `netstandard2.0`.
+
+You can override the minimum runtime patch version (to higher or lower versions) in the project file, as shown in the following example:
```

Comment:
It feels like an abrupt jump to start talking about `RuntimeFrameworkVersion` here.  Perhaps this belongs later in the document (after the roll forward and possibly self-contained discussion?)

Reply:
It's also important to note that `RuntimeFrameworkVersion` does not specify a minimum patch version, at least for self-contained apps (which is probably the main scenario for using it).  For self-contained apps, the exact version specified will be used, even if there is a later patch version available.

Reply:
I moved this section to the end of the article.